duckduckgo/iOS
# use new WKWebsiteDataStore

Task/Issue URL: https://app.asana.com/0/0/1205887954649226/f
Tech Design URL:
CC:

**Description**:
Move to use new WKWesbiteDataStore API for creating a persistent container. 

**Steps to test this PR**:

Using a device:
1. Using `develop` sign into at least one website (e.g. Netflix) and fireproof it 
2. Use https://www.w3schools.com/js/tryit.asp?filename=tryjs_cookie_username to set a cookie
3. Open a new tabs on various websites
4. Check out this branch and run the app
5. Check you're still logged into the website, the cookie is still set and all the tabs are still present
6. Use the fire button 
7.  Check you're still logged into the website, but the cookie has been cleared
8. Use the fire button again
9. Open and close some tabs
10. Using the tab switcher add a new tab and use the fire button
11. Remove the domain from the list of fire proofed sites (in settings) and refresh the webview with that site, you should now be logged out

Using the simulator:
1. Check that the containers are deleted from disk
* see Xcode console for location of app on disk using the simulator
* add a breakpoint to `DataStoreIdManager` in `WKWebViewConfigurationExtension.swift` to get the container id.  
* Note that the containerId on disk is in **lowercase**.  
* Use a command like `find <path to app> | grep <container id lowercased>` before and after the fire button
